Automattic/wp-calypso
# Cart: Add support for quantities

#### Changes proposed in this Pull Request

This PR adds support for the `quantity` field in the cart.

#### Testing instructions

* Add a product to the cart and specify the quantity, e.g. `addItem( { product_id: 400, quantity: 5 } )`
* Verify the price = `quantity * price per item`
